lxc/init.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/spf13/cobra"
 	"gopkg.in/yaml.v2"
 
-	"github.com/canonical/lxd/client"
+	lxd "github.com/canonical/lxd/client"
```

Comment:
@boltmark please can you not auto complete the `client` package as `lxd` here. Thanks

Reply:
Oops, fixed